### PR TITLE
OpenAI PR Agent Setup

### PR DESCRIPTION
**Pull request by somesylvie**
### **OpenAI PR Agent Setup**
- Added a new PR agent AI reviewer.
- Our model is using ChatGPT-4 currently as it was recommended as a good text reading LLM. We can always use a different LLM with Azure OpenAI or run different LLMs as a back up.
- Using mostly default settings, PR Agent is set to Auto-Review/Auto-Improve.
  - We removed auto-describe as we want to encourage engineers to review their stories as part of the PR review process when writing up a PR Description. 
- See https://qodo-merge-docs.qodo.ai/tools/ for all the manual triggering features.
- See https://qodo-merge-docs.qodo.ai/usage-guide/configuration_options/ for all configuration options.
  - Config was added in [another PR](https://github.com/CDCgov/trusted-intermediary/pull/1430) because the agent only reads config files that are merged into main

## Issue

https://github.com/CDCgov/trusted-intermediary/issues/1357

### CODE REVIEW

File: .github/workflows/pr_agent.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Perry the PR Agent Platypus
```
Comment (somesylvie) [inline .github/workflows/pr_agent.yml:1]: /review

Comment (somesylvie) [reply .github/workflows/pr_agent.yml:1]: /describe

Comment (somesylvie) [reply .github/workflows/pr_agent.yml:1]: /review

Comment (somesylvie) [reply .github/workflows/pr_agent.yml:1]: /describe

**Review by saquino0827 — APPROVED** [review]

**Review by pluckyswan — APPROVED** [review]

**Pull request merged**